docs/contributing/index.rst

```diff
@@ -63,11 +63,75 @@ And now you can use gulp for installing views, by just running a simple command:
 
     $ yarn run gulp
 
-For the contributing process questions, please refer to the `Contributing Guide <http://docs.sylius.org/en/latest/contributing/index.html>`_ that comes up in the following chapters.
+For the contributing process questions, please refer to the `Contributing Guide <http://docs.sylius.org/en/latest/contributing/index.html>`_ that comes up in the following chapters:
+
+.. toctree::
+    :maxdepth: 1
+
+    code/index
+    documentation/index
+    translations/index
 
 .. _Gulp: http://gulpjs.com/
 .. _Node.js: https://nodejs.org/en/download/
 
+
+How to contribute using Sylius Vagrant
+--------------------------------------
+
+.. warning::
+
+    This article assumes you're familiar with `Composer`_, a dependency manager
+    for PHP. It also assumes you have `Composer installed globally`_.
+    Basic knowledgle about `Vagrant <https://www.vagrantup.com/about.html>`_ is also required,
```

Comment:
knowledgle -> knowledge

Reply:
@GSadee Thank you, i've also updated on this page:http://docs.sylius.org/en/latest/book/installation/vagrant_installation.html.